application/org.openjdk.jmc.flightrecorder.flameview/src/main/java/org/openjdk/jmc/flightrecorder/flameview/FlameGraphJSONMarshaller.java

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
Normally we use initial capital case and the rest lower case for both acronyms and initialisms. You may see exceptions in the code base, since (IIRC) we used to have a different ruler separating them, and also taking into account how long they were, but eventually I believe we settled on initial capital case, the rest lower to keep things simple.

Reply:
This is true for methods as well - will not add this to all places.

Reply:
I got this habit from Go, the style guide specifically states that all acronyms should be uppercase.

It took a while to get used to but I started liking it afterwards 😃

Reply:
That's how we started out. That said, the uppercase completion and search in IDEs makes it so very convenient to go with uppercase first then lower case. It also avoid some cases where it becomes a bit unreadable, such as the last letter of the acronym being the same as the first letter of the next part. Using the same for both acronyms and initialisms makes the rule easier to understand and explain.